src/TwigComponent/src/Command/ComponentDebugCommand.php

```diff
@@ -0,0 +1,214 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\TwigComponent\Command;
+
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Helper\Table;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Finder\Finder;
+use Symfony\UX\TwigComponent\ComponentFactory;
+use Symfony\UX\TwigComponent\Twig\PropsNode;
+use Twig\Environment;
+
+#[AsCommand(name: 'debug:component', description: 'Display current components and them usages for an application')]
+class ComponentDebugCommand extends Command
+{
+    public function __construct(private string $twigTemplatesPath, private ComponentFactory $componentFactory, private Environment $twigEnvironment, private iterable $components)
+    {
+        parent::__construct();
+    }
+
+    protected function configure(): void
+    {
+        $this
+            ->setDefinition([
+                new InputArgument('name', InputArgument::OPTIONAL, 'A component name'),
+            ])
+            ->setHelp(<<<'EOF'
+                The <info>%command.name%</info> display current components and them usages for an application:
+
+                  <info>php %command.full_name%</info>
+
+                EOF
+            )
+        ;
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $io = new SymfonyStyle($input, $output);
+        $name = $input->getArgument('name');
+
+        if ($name) {
+            try {
+                $metadata = $this->componentFactory->metadataFor($name);
+            } catch (\Exception $e) {
+                $io->error($e->getMessage());
+
+                return Command::FAILURE;
+            }
+
+            $class = $metadata->get('class');
+            $allProperties = [];
+
+            if ($class) {
```

Comment:
I think it can be useful to display: 
- properties with the ExposeInTemplate attribute, and if the name is set show the name.
- if the component has a mount, premount, and postmount hooks

And for LiveConponents:
- as @kbond said show if the components is live
- show live props
- show live actions

Reply:
Hi @WebMamba and @kbond, an update with `Events` and `LiveAction`, it could be look like this for a Live component: 
 
```
 $ php bin/console debug:component RandomNumber
+--------------------+-----------------------------------+
| Property           | Value                             |
+--------------------+-----------------------------------+
| Component          | RandomNumber                      |
| Type               | AsLiveComponent                   |
| Class              | App\Components\RandomNumber       |
| Template           | components/RandomNumber.html.twig |
| Properties         | string $name = toto               |
|                    | string $type = test               |
| Live Properties    | int $max = 1000                   |
|                    | int $min = 10                     |
| Events             | Mount                             |
|                    | PreMount                          |
| LiveAction Methods | resetMax                          |
+--------------------+-----------------------------------+
```

And for a TwigComponent

```
$ php bin/console debug:component Coucou
+------------+-----------------------------+
| Property   | Value                       |
+------------+-----------------------------+
| Component  | Coucou                      |
| Type       | AsTwigComponent             |
| Class      | App\Components\Alert        |
| Template   | components/Coucou.html.twig |
| Properties | string $type = success      |
|            | string $message             |
| Events     | Mount                       |
|            | PreMount                    |
+------------+-----------------------------+
```

But for the `#[ExposeInTemplate]` what could I render for you? 
The name of the property in the class or the one one displayed in the template? And with the getter or not?